wasmtime-obj/src/function.rs

```diff
@@ -4,12 +4,36 @@ use cranelift_entity::EntityRef;
 use faerie::{Artifact, Decl, Link};
 use wasmtime_environ::{Compilation, Module, RelocationTarget, Relocations};
 
+fn get_reloc_target_special_import_name(target: RelocationTarget) -> Option<&'static str> {
+    Some(match target {
+        RelocationTarget::Memory32Grow => &"_wasm_memory32_grow",
```

Comment:
Prefixing these with "_" is specific to Darwin; Linux doesn't add "_" prefixes. Also, currently these functions are named "wasmtime_*". I could be ok changing them, but either way it'd be good to match them.

Reply:
I don't have a preference, except to make a unique name. Renamed with "wasmtime_*" prefix.